pkg/dataMover/data_mover.go

```diff
@@ -101,6 +112,13 @@ func (this *DataMover) CopyToRepo(peID astrolabe.ProtectedEntityID) (astrolabe.P
 	this.RegisterOngoingUpload(peID, cancelFunc)
 
 	log.Debugf("Ready to call s3 PETM copy API for local PE")
+	if backupRepository.Name != "Not use" {
+		s3PETM, err := utils.GetRepositoryFromBackupRepository(backupRepository, log)
```

Comment:
Although `GetRepositoryFromBackupRepository` is not part of this change, would it be possible to rename it to `GetS3RepositoryFromBackupRepository`, I think it would help anyone reading the code.

Reply:
Let's change s3PETM to repoPETM.  We may use something besides S3 here

Reply:
> Although `GetRepositoryFromBackupRepository` is not part of this change, would it be possible to rename it to `GetS3RepositoryFromBackupRepository`, I think it would help anyone reading the code.

As we may use some other repo in the future, so I think the general name GetRepositoryFromBackupRepository may be better.